FreeRTOS/FreeRTOS-Plus-TCP
# Add Unit Test for FreeRTOS_IPv6_Utils

Add UT for FreeRTOS_IPv6_Utils

Description
-----------
As the FreeRTOS_IPv6_Utils was added for merging IPv4 and IPv6, this change adds the Unit test case for this new file along with 100% coverage.

Test Steps
-----------
Unit Test
```
cmake -S test/unit-test -B test/unit-test/build/ -DCMAKE_BUILD_TYPE=Debug
make -C test/unit-test/build/ all
```
Coverage Test
```
make -C ${BUILD_DIR} coverage
lcov --list --rc lcov_branch_coverage=1 ${BUILD_DIR}coverage.info
```

Checklist:
----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
